packages/server/tests/acceptance/index.spec.ts

```diff
@@ -129,6 +129,7 @@ describe('RPC Server Acceptance Tests', function () {
       RELAY_URL,
       CHAIN_ID,
       Utils.generateRequestId(),
+      Number(process.env.INITIAL_ALIAS_ACCOUNT_AMOUNT_IN_HBAR || 2000),
```

Comment:
In `configuration.md` file it is documented as `TEST_INITIAL_ACCOUNT_STARTING_BALANCE`, but here we use `INITIAL_ALIAS_ACCOUNT_AMOUNT_IN_HBAR`

Reply:
Updated.